### ISSUE TITLE
[Accessibility], WCAG 1.4.3. The contrast of the date and time in the News section is too low

### ISSUE THREAD

**Issue by Eiskore** [NatLibFi/Skosmos#1964]
## URL address of the page where you encountered the problem
https://test.dev.finto.fi/fi/

## Description of the problem
The News section has grey text on a light grey background. Element has insufficient color contrast of 2.24 (foreground color: #999999, background color: #dfe5ed,). Expected contrast ratio is at least 4.5:1
Severity: Low
Priority: Medium 
Fix: Use darker foreground color, eg. #626262 and the original background color: #dfe5ed to meet a contrast ratio of 4.81:1.

## Additional information (e.g. screenshots) about the problem

<img width="838" height="248" alt="Image" src="https://github.com/user-attachments/assets/86d9c0d6-d841-4c05-9a8c-b17893633e93" />

## The browser you used when the problem appeared
Chrome, Firefox, Edge

**Comment by osma** [comment]
Fixed with the redesign of the news section in this Finto-data commit: https://github.com/NatLibFi/Finto-data/commit/4a18be6c0b9c0ef70566f6a077f784a3a84ce1bf

**Comment by tinskuu** [comment]
The issue is fixed.
